### PR TITLE
SUBMARINE-288.Set some users to have root privilege

### PR DESCRIPTION
**Pull request by huiyangjian**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - https://submarine.apache.org/contribution/contributions.html

### What type of PR is it?
[Bug Fix | Improvement | Feature | Documentation | Hot Fix | Refactoring]

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/SUBMARINE/
* Put link here, and add [SUBMARINE-*Jira number*] in PR title, eg. [SUBMARINE-23]

### How should this be tested?
* First time? Setup Travis CI as described on https://submarine.apache.org/contribution/contributions.html#continuous-integration
* Strongly recommended: add automated unit tests for any new or changed behavior
* Outline any manual steps to test the PR here.

### Questions:
* Does the licenses files need update? Yes/No
* Is there breaking changes for older versions? Yes/No
* Does this needs documentation? Yes/No

### CODE REVIEW

File: docs/ecosystem/kaldi/sge/sge_run.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
```
Comment (jiwq) [inline docs/ecosystem/kaldi/sge/sge_run.sh]: missing license

File: submarine-server/server-submitter/submitter-yarnservice/src/main/java/org/apache/submarine/server/submitter/yarnservice/utils/EnvironmentUtilities.java [hunk 2]
```diff
@@ -120,4 +125,25 @@ static void appendToEnv(Service service, String key, String value,
...
+
+  static void etcAppendToEnv(Service service, String key, String value,
+                             String delim) {
+    Map<String, String> env = service.getConfiguration().getEnv();
+    if (!env.containsKey(key)) {
+      env.put(key, value);
+    } else {
+      if (!value.isEmpty()) {
+        String existingValue = env.get(key);
+        if((existingValue.contains(ETC_PASSWD) && value.contains(ETC_PASSWD))
```
Comment (jiwq) [inline submarine-server/server-submitter/submitter-yarnservice/src/main/java/org/apache/submarine/server/submitter/yarnservice/utils/EnvironmentUtilities.java]: checkstyle

Comment (huiyangjian) [reply submarine-server/server-submitter/submitter-yarnservice/src/main/java/org/apache/submarine/server/submitter/yarnservice/utils/EnvironmentUtilities.java]: thanks

**Comment by liuxunorg** [comment]
@huiyangjian Please fixed this issue in you code, Thanks!

```
[ERROR] src/main/java/org/apache/submarine/server/submitter/yarnservice/utils/EnvironmentUtilities.java:[0] (misc) NewlineAtEndOfFile: File does not end with a newline.
[ERROR] src/main/java/org/apache/submarine/server/submitter/yarnservice/utils/EnvironmentUtilities.java:[137,11] (whitespace) WhitespaceAround: 'if' is not followed by whitespace.
[ERROR] src/main/java/org/apache/submarine/server/submitter/yarnservice/utils/EnvironmentUtilities.java:[138] (indentation) Indentation: '||' have incorrect indentation level 11, expected level should be 12.
```

**Comment by liuxunorg** [comment]
https://api.travis-ci.org/v3/job/615480341/log.txt
```
 !????? docs/ecosystem/kaldi/sge/group
  AL    docs/ecosystem/kaldi/sge/sge_run.sh
 !????? docs/ecosystem/kaldi/sge/passwd
```
Because `docs/ecosystem/kaldi/sge/group` and `docs/ecosystem/kaldi/sge/passwd` not have licenses head, So, you can exclude rat check this two file in there:
https://github.com/huiyangjian/submarine/blob/master/pom.xml#L431

**Comment by liuxunorg** [comment]
@huiyangjian Please fixed this issue in you code, Thanks!
https://api.travis-ci.org/v3/job/615580062/log.txt
```
[ERROR] /home/travis/build/apache/submarine/submarine-server/server-submitter/submitter-yarnservice/src/main/java/org/apache/submarine/server/submitter/yarnservice/utils/EnvironmentUtilities.java:[103,9] cannot find symbol
  symbol:   variable ETC_GROUP_MOUNT_STRING
  location: class org.apache.submarine.server.submitter.yarnservice.utils.EnvironmentUtilities
```

**Pull request closed without merging**